startup_scripts/210_dcim_interfaces.py

```diff
@@ -29,6 +36,35 @@
     interface, created = Interface.objects.get_or_create(**matching_params, defaults=defaults)
 
     if created:
-        print("🧷 Created interface", interface.name, interface.device.name)
+        print(f"🧷 Created interface {interface} on {interface.device}")
 
     set_custom_fields_values(interface, custom_field_data)
+
+    for related_field, related_value in related_interfaces.items():
+        if not related_value:
+            continue
+
+        r_model, r_field = related_assocs[related_field]
+
+        if related_field == "parent" and not interface.parent_id:
+            query = {r_field: related_value, "device": interface.device}
+            try:
+                related_obj = r_model.objects.get(**query)
+            except Interface.DoesNotExist:
+                print(f"⚠️ Could not find parent interface with: {query} for interface {interface}")
+                raise
+
+            interface.parent_id = related_obj.id
+            interface.save()
+            print(
+                f"🧷 Attached interface {interface} on {interface.device} "
+                f"to parent {related_field}"
```

Comment:
`related_field` is always parent here. I think this should be `related_obj` (name (preferred) or id).

Reply:
Thanks. Corrected.